cockatrice/src/player.cpp

```diff
@@ -1896,9 +1898,13 @@ void Player::addCard(CardItem *c)
 void Player::deleteCard(CardItem *c)
 {
     if (dialogSemaphore)
```

Comment:
If you want to check for `c` to be not null, better do it before the `if`, since both branches needs `c` ito be a valid pointer.

Reply:
addressed